src/routes/console/project-[project]/auth/teams/team-[team]/updatePrefs.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+    import { invalidate } from '$app/navigation';
+    import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import { CardGrid, Heading } from '$lib/components';
+    import { Dependencies } from '$lib/constants';
+    import { Button, Form } from '$lib/elements/forms';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdk } from '$lib/stores/sdk';
+    import { onMount } from 'svelte';
+    import { team } from './store';
+
+    $: if (prefs) {
+        if (JSON.stringify(prefs) !== JSON.stringify(Object.entries($team.prefs))) {
```

Comment:
@TorstenDittmann now that we have the `difference` method for arrays we can use that instead of this right? :thinking:

Reply:
its an object, so I am not sure 🤔